boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+#
+# Common script to generate OSD operator bundles for publishing to OLM. Copies appropriate files
+# into a directory, and composes the ClusterServiceVersion which needs bits and
+# pieces of our rbac and deployment files.
+#
+# Usage ./common-generate-operator-bundle.py -o OPERATOR_NAME -d OUTPUT_DIR -p PREVIOUS_VERSION -n GIT_NUM_COMMITS -c COMMIT_HASH -i HIVE_IMAGE
+
+import datetime
+import os
+import sys
+import yaml
+import shutil
+import argparse
+import string
+
+# This script will append the current number of commits given as an arg
+# (presumably since some past base tag), and the git hash arg for a final
+# version like: 0.1.189-3f73a592
+VERSION_BASE = "0.1"
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-o", "--operator-name", type=str, help="Name of the operator", required=True)
+parser.add_argument("-d", "--output-dir", type=str, help="Directory for the CSV generation", required=True)
+parser.add_argument("-p", "--previous-version", type=str, help="Directory for the CSV generation", required=True)
+parser.add_argument("-n", "--commit-number", type=str, help="Number of commits in the project (used for version generation)", required=True)
+parser.add_argument("-c", "--commit-hash", type=str, help="Current commit hashDirectory for the CSV generation (used for version generation)", required=True)
+parser.add_argument("-i", "--operator-image", type=str, help="Base index image to be used", required=True)
+args = parser.parse_args()
+
+operator_name   = args.operator_name
+outdir          = args.output_dir
+prev_version    = args.previous_version
+git_num_commits = args.commit_number
+git_hash        = args.commit_hash
+operator_image  = args.operator_image
+
+full_version = "%s.%s-%s" % (VERSION_BASE, git_num_commits, git_hash)
+print("Generating CSV for version: %s" % full_version)
+
+if not os.path.exists(outdir):
+    os.mkdir(outdir)
+
+version_dir = os.path.join(outdir, full_version)
+if not os.path.exists(version_dir):
+    os.mkdir(version_dir)
+
+with open('config/templates/csv-template.yaml'.format(operator_name), 'r') as stream:
```

Comment:
Did you mean to use a substitution placeholder (`{}`) in the format string somewhere? As written, the `.format(operator_name)` part of this is a no-op.

[Later] Based on what I see elsewhere, I think you meant to leave out the operator name bit.
```suggestion
with open('config/templates/csv-template.yaml', 'r') as stream:
```